### ISSUE TITLE
Tibiotalar (ankle) arthritis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11214]
**Preferred term label:**
Tibiotalar (ankle) arthritis

**Synonyms**

**Definition (free text, please give PubMed ID)**
AP, oblique, or lateral view x-ray of the ankle showing radiographic evidence of osteoarthritis such as joint space narrowing, subchondral sclerosis, subchondral cysts, and/or osteophyte formation in the tibiotalar joint

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ankle arthritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001331
- label: Tibiotalar arthritis
- parents: [Abnormal ankle morphology (HP:0034673)](https://hpo.jax.org/app/browse/term/HP:0034673)
- pmid: PMID:26660651
- def: AP, oblique, or lateral view x-ray of the ankle showing radiographic evidence of osteoarthritis such as joint space narrowing, subchondral sclerosis, subchondral cysts, and/or osteophyte formation in the tibiotalar joint.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11214
- synonyms: Ankle arthritis(NONE)
